backend/hct_mis_api/apps/household/fixtures.py

```diff
@@ -423,33 +423,38 @@ def create_household_and_individuals(
     if household_data.get("size") is None:
         household_data["size"] = len(individuals_data)
     if "program" not in household_data:
-        program = ProgramFactory()
-    else:
-        program = household_data["program"]
-    if "registration_data_import" not in household_data:
-        rdi = RegistrationDataImportFactory(program=program)
-        household_data["registration_data_import"] = rdi
+        household_data["program"] = ProgramFactory()
```

Comment:
I think we have to keep this logic with `household_data["program"] and household_data["registration_data_import"]`